README.md

```diff
@@ -9,25 +9,75 @@ The tempest snap can be installed directly from the snap store:
+- designate-tempest-plugin
+- ec2api-tempest-plugin
+- freezer-tempest-plugin
+- glance-tempest-plugin
+- heat-tempest-plugin
+- ironic-tempest-plugin
+- keystone-tempest-plugin
+- kuryr-tempest-plugin
+- magnum-tempest-plugin
+- manila-tempest-plugin
+- mistral-tempest-tests
+- monasca-tempest-plugin
+- murano-tempest-plugin
+- neutron-tempest-plugin
+- octavia-tempest-plugin
+- oswin-tempest-plugin
+- sahara-tests
+- senlin-tempest-plugin
+- solum-tempest-plugin
+- telemetry-tempest-plugin
+- trove-tempest-plugin
+- venus-tempest-plugin
+- vitrage-tempest-plugin
+- watcher-tempest-plugin
+- zaqar-tempest-plugin
+- zun-tempest-plugin
+
 
 ## Quickstart
-To run Tempest, please follow the steps in [Tempest QuickStart](https://docs.openstack.org/tempest/latest/overview.html#quickstart) documentation.
+To run Tempest, please follow the steps in [Tempest QuickStart] documentation.
 
-The tempest snap provides a set of pre-defined test lists with different focuses. For quickly verifying the behavior of the objective cloud and making sure tempest is correctly set up, you can run tests listed in the `readonly-quick.txt` file:
+The tempest snap provides a set of pre-defined test lists with different focuses.
+For quickly verifying the behavior of the objective cloud and making sure tempest
+is correctly set up, you can run tests listed in the `readonly-quick.txt` file:
 
     tempest run --load-list '@BUILTIN_TESTLISTS/readonly-quick.txt' 
 
-`@BUILTIN_TESTLISTS` keyword will be automatically replaced by the full path of the directory storing the built-in test lists.
+`@BUILTIN_TESTLISTS` keyword will be automatically replaced by the full path of
+the directory storing the built-in test lists.
 
-**Note**: the single-quotation marks around the test list file path is required if `@BUILTIN_TESTLISTS` keyword is used (in another words, when running tempest with a built-in test list file). Fail to do that will result in wrong path error.
+**Note**: the single-quotation marks around the test list file path is required
+if `@BUILTIN_TESTLISTS` keyword is used (in another words, when running tempest
+with a built-in test list file). Fail to do that will result in wrong path error.
 
 
 ## This snap includes
 
 ### python-tempestconf
 
-This snapshot includes [python-tempestconf](https://opendev.org/openinfra/python-tempestconf),
-which automatically generates a tempest configuration based on your cloud.
+This snapshot includes [python-tempestconf], which automatically generates a tempest
+configuration based on your cloud.
+
+It provided single CLI options [`discover-tempest-config`] which is provided in snap
+via `tempest.discover`.
 
-It provided single CLI options [`discover-tempest-config`](https://docs.opendev.org/openinfra/python-tempestconf/latest/cli/cli_options.html#discover-tempest-config),
-which is provided in snap via `tempest.discover`.
+[OpenStack release notes]: https://releases.openstack.org/antelope/index.html#tempest-plugins
```

Comment:
too bad there's no `https://releases.openstack.org/latest/index.html#tempest-plugins`...

Reply:
Yes, unfortunately we have to update this each time a new os release is out.